setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+import bus_sdk
+
+setup(
+    name='bus_sdk',
```

Comment:
not `bussdk` due to https://www.python.org/dev/peps/pep-0008/#package-and-module-names ? Not particularly readable

Reply:
Yes, it was the readability that was the issue. Even though its discouraged in PEP8 underscores are still used in major package names to improve readability. An example I use a lot is http://www.django-rest-framework.org

usage:
`from rest_framework import routers, serializers`

I will always avoid '_' where possible but I think it is fine in this use case.